templates/_helpers.tpl

```diff
@@ -115,3 +115,23 @@ Calculate provisoner's bcrypt-hashed password
 {{- define "graphdb.security.provisioner.passwordHash" -}}
   {{- printf "%s" ( htpasswd .Values.security.provisioner.username .Values.security.provisioner.password | trimPrefix (printf "%s:" .Values.security.provisioner.username)) -}}
 {{- end -}}
+
+{{/*
+Render string and yaml templates
+*/}}
+{{- define "render.list.singleMaps" -}}
```

Comment:
What are the advantages of this over the approach in https://github.com/Ontotext-AD/graphdb-helm/blob/main/templates/extra-objects.yaml#L3